app/decorators/physical_switch_decorator.rb

```diff
@@ -0,0 +1,42 @@
+class PhysicalSwitchDecorator < MiqDecorator
+  def self.fonticon
+    'ff ff-network-switch'
+  end
+
+  def quadicon
+    {
+      :top_left     => {
+        :text    => (hardware.physical_ports ? hardware.physical_ports.size : 0),
+        :tooltip => 'Ports'
+      },
+      :top_right    => {
+        :tooltip => power_state.try(:downcase),
+      }.merge(QuadiconHelper.machine_state(power_state)),
+      :bottom_left  => {
+        :fonticon => fonticon,
+        :tooltip  => type,
+      },
+      :bottom_right => {
+        :fileicon => health_state_img,
+        :tooltip  => health_state,
+      }
+    }
+  end
+
+  def single_quad
+    {
+      :fonticon => fonticon
+    }
```

Comment:
I think you're missing the tooltip here.

Reply:
Done. I will update in the next minutes.